### PR TITLE
added in private and certificate for lets encrypt signed

### PR DESCRIPTION
**Pull request by moskalyk**
this will make anyone who goes to bablot.ca https encrypted

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -27,8 +27,12 @@ http {
...
           access_log  off;
 
-          ssl_certificate     /etc/nginx/ssl/certificate.pem;
-          ssl_certificate_key /etc/nginx/ssl/private.key;
+          #ssl_certificate     /etc/nginx/ssl/certificate.pem;
+          #ssl_certificate_key /etc/nginx/ssl/private.key;
+
+          ssl_certificate       /etc/nginx/ssl/certificate.pem;
+          ssl_certificate_key   /etc/nginx/ssl/private.key;
+
```
Comment (dharness) [inline nginx/nginx.conf]: do we need the commented stuff?

File: nginx/ssl/certificate.pem [hunk 2]
```diff
@@ -1,16 +1,56 @@
 -----BEGIN CERTIFICATE-----
```
Comment (dharness) [inline nginx/ssl/certificate.pem]: once we confirm that this works are we going to redo and ecrypt all this, or get it on the server by some other means

File: nginx/nginx.conf (hunk 1 above)
Comment (moskalyk) [inline nginx/nginx.conf]: removed

**Comment by dharness** [comment]
LGTM

**Pull request merged**